packages/playwright/src/util.ts

```diff
@@ -206,8 +206,7 @@ export function addSuffixToFilePath(filePath: string, suffix: string): string {
   return base + suffix + ext;
 }
 
-export function sanitizeFilePathBeforeExtension(filePath: string): string {
-  const ext = path.extname(filePath);
+export function sanitizeFilePathBeforeExtension(filePath: string, ext = path.extname(filePath)): string {
```

Comment:
For some reason, I feel uncomfortable with non-constant default values. I'd rather see any code evaluated during the function call inside the function body.